src/include/omvll/passes/string-encoding/Routines.h

```diff
@@ -0,0 +1,8 @@
+namespace omvll {
+
+using EncRoutineFn = void(char *, const char *, unsigned long long, int);
+EncRoutineFn *getEncodeRoutine(unsigned Idx);
+const char *getDecodeRoutine(unsigned Idx);
+unsigned getNumEncodeDecodeRoutines();
+
+} // namespace omvll
```

Comment:
```suggestion
} // end namespace omvll
```

Reply:
The current code is what clang-format version 20 proposed. Should I still add your proposal on top?

Reply:
Right, yes – I'd prefer to follow the existing style in the codebase.